### ISSUE TITLE
Wallet.Create with authId and authSecret results in type error 

### ISSUE THREAD

**Issue by Schwartz10** [3box/identity-wallet-js#71]
```
      const ceramic = new CeramicClient();
      const idWallet = await IdentityWallet.create({
        authId: 'randomString',
        authSecret: nacl.randomBytes(32),
        getPermission: () => Promise.resolve([]),
        ceramic,
      });
```

results in:

```TypeError: Cannot read property 'doctype' of undefined```

Identity wallet version `2.0.0-alpha.3`
Ceramic http client version `0.5.11`

**Comment by oed** [comment]
Thanks, will try to replicate this.

**Comment by oed** [comment]
Should be fixed in `identity-wallet@2.0.0-alpha.4`
